docs/running_Occlum_with_Docker_and_OCI_Runtime_rune.md

```diff
@@ -0,0 +1,140 @@
+# Running Occlum with Docker and OCI Runtime rune
+
+This user guide provides the steps to run Occlum with Docker and OCI Runtime `rune`.
+
+[rune](https://github.com/alibaba/inclavare-containers/tree/master/rune) is a novel OCI Runtime used to run trusted applications in containers with the hardware-assisted enclave technology.
+
+[Occlum](https://github.com/occlum/occlum) is a memory-safe, multi-process library OS for Intel SGX.
+
+# Requirements
+
+- Ensure that you have one of the following required operating systems to build a Occlum container image:
+  - CentOS 8.1
+  - Ubuntu 18.04-server
+
+- Please follow [Intel SGX Installation Guide](https://download.01.org/intel-sgx/sgx-linux/2.11/docs/Intel_SGX_Installation_Guide_Linux_2.11_Open_Source.pdf) to install Intel SGX driver, Intel SGX SDK & PSW for Linux.
+  - For CentOS 8.1, UAE service libraries are needed but may not installed if SGX PSW installer is used. Please manually install it:
+    ```shell
+    rpm -i libsgx-uae-service-2.11.100.2-1.el8.x86_64.rpm
+    ```
+
+- Install [enable_rdfsbase kernel module](https://github.com/occlum/enable_rdfsbase#how-to-build), allowing to use FSGSBASE instructions in Occlum.
```

Comment:
If user is using 5.9 kernel, user does not need this module. Some kernel disabled this feature, even user installed this driver, user still not able to run Occlum. We could add a hint.